arelle/XmlValidateParticles.py

```diff
@@ -11,7 +11,12 @@
 from arelle.ModelObject import ModelObject, ModelAttribute
 from arelle.XmlValidate import validate
 
+BUILTIN_XML_SCHEMA = 'http://www.w3.org/2001/XMLSchema.xsd'
+
+
 def validateElementSequence(modelXbrl, compositor, children, ixFacts, iNextChild=0):
+    if compositor.modelDocument.uri == BUILTIN_XML_SCHEMA:
```

Comment:
This is still checking the document URL rather than the XSD target namespace.  i.e., it seems like this should be checking `targetNamespace` against `XbrlConst.xsd`.